flow/src/main/java/com/sequenceiq/flow/service/FlowService.java

```diff
@@ -195,9 +196,23 @@ public FlowCheckResponse getFlowState(String flowId) {
         flowCheckResponse.setFlowId(flowId);
         flowCheckResponse.setHasActiveFlow(!completed("Flow", flowId, List.of(), allByFlowIdOrderByCreatedDesc));
         flowCheckResponse.setLatestFlowFinalizedAndFailed(isFlowInFailedState(allByFlowIdOrderByCreatedDesc, failHandledEvents));
+        setStateEventFlowTypeOnFlowCheckResponse(flowCheckResponse, allByFlowIdOrderByCreatedDesc, "flow", flowId);
         return flowCheckResponse;
     }
 
+    public void setStateEventFlowTypeOnFlowCheckResponse(FlowCheckResponse flowCheckResponse, List<FlowLogWithoutPayload> relatedFlowLogs,
+        String marker, String id) {
```

Comment:
@lturcsanyi Thank you!! Im not 100% sure where we want to align this too since if I use IntelliJ reformat, it will be aligned with "FlowcheckResponse", the first parameter, which looks weird. And I checked other codes in the repo it seems everyone is doing this differently. So I changed it to be aligned with "public". If it still needs changes please let me know. Again, TYSM for reviewing!

Reply:
There may be a small gap in the formatter, but if you use the CB formatter [https://github.com/hortonworks/cloudbreak#import-project](https://github.com/hortonworks/cloudbreak#import-project) you can just hit enter after  `List<FlowLogWithoutPayload> relatedFlowLogs,` it should indent the new line correctly.


> And I checked other codes in the repo it seems everyone is doing this differently

Hopefully its not true. Good example from this class: `FlowService.fetchFlowLogsByParentChainId`

Reply:
@lturcsanyi Thanks a lot for this detailed instruction! I imported the CB formatter and it works great. Please let me know if the new indentation looks good to you. Thank you!!